src/process/exec_linux.go

```diff
@@ -31,9 +31,11 @@ func (e *Executor) ExecCommand(sandbox bool, command string, args ...string) *ex
 			// Otherwise exec the sandbox tool
 			args = append([]string{command}, args...)
 			command = e.sandboxTool
+			env = []string{"SANDBOX_NETWORK=" + boolToString(sandbox.Network), "SANDBOX_MOUNT=" + boolToString(sandbox.Mount)}
 		}
 	}
 	cmd := exec.Command(command, args...)
```

Comment:
We could probably just move this above the if statement and get rid of the intermediate `env` variable

Reply:
Ya, the PR needs tidying up. Thanks